types/react-native-mixpanel/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for react-native-mixpanel 0.1
+// Project: https://github.com/davodesign84/react-native-mixpanel#readme
+// Definitions by: My Self <https://github.com/me>
```

Comment:
Are you sure you don't want to add yourself as the author? Not having an author makes future updates take a lot longer.

Reply:
Oops. My fail.

Reply:
Updated and fixed it.